src/ui/actions/timeline.ts

```diff
@@ -264,6 +269,44 @@ export function seek(
   };
 }
 
+export function jumpToReduxCode(
+  point: ExecutionPoint,
+  time: number,
+  sourcesState: SourcesState
+): UIThunkAction {
+  return async (dispatch, _, { replayClient }) => {
+    dispatch(framePositionsCleared());
+
+    const pauseId = await pauseIdCache.readAsync(replayClient, point, time);
+    const frames = (await getPauseFramesAsync(replayClient, pauseId, sourcesState)) ?? [];
+    const filteredPauseFrames = frames.filter(frame => {
+      const { source } = frame;
+      if (!source) {
+        return false;
+      }
+
+      // Filter out everything in `node_modules`, so we have just app code left
+      // TODO There may be times when we care about renders queued by lib code
+      // TODO See about just filtering out React instead?
+      return !MORE_IGNORABLE_PARTIAL_URLS.some(partialUrl => source.url?.includes(partialUrl));
+    });
+
+    const frameSteps = await frameStepsCache.readAsync(
+      replayClient,
+      pauseId,
+      // The first 2 elements in filtered pause frames are from replay's redux stub, so they should be ignored
```

Comment:
:question: Can you paste in an example of what the callstack looks like here?

Reply:
Do you mean the filtered pause frames?